packages/flutter_adaptive_scaffold/lib/src/breakpoints.dart

```diff
@@ -30,51 +30,154 @@ class Breakpoints {
   static const Breakpoint standard = WidthPlatformBreakpoint(begin: -1);
 
   /// A window whose width is less than 600 dp and greater than 0 dp.
-  static const Breakpoint small = WidthPlatformBreakpoint(begin: 0, end: 600);
+  static const Breakpoint small = WidthPlatformBreakpoint(
```

Comment:
Are these changes going to affect existing users of these breakpoints?

I worry that it will cause unforeseen changes in existing applications.

Or is this just defining new top/bottom criteria that didn't exist before?  What were the implicit top/bottom before? Were they unbounded?

Reply:
It is new criteria, the old behaviour was unbounded. This will only affect some edge cases like foldables or extreme form factors. Even then it will probably improve the situation because with this it will switch to a different navigation which is more appropriate for that form factor.